src/lib.rs

```diff
@@ -29,30 +32,48 @@ use raw_window_handle::{
 
 /// An instance of this struct contains the platform-specific data that must be managed in order to
 /// write to a window on that platform.
-pub struct GraphicsContext {
+pub struct Context {
     /// The inner static dispatch object.
     ///
-    /// This is boxed so that `GraphicsContext` is the same size on every platform, which should
+    /// This is boxed so that `Context` is the same size on every platform, which should
     /// hopefully prevent surprises.
-    graphics_context_impl: Box<Dispatch>,
+    context_impl: Box<ContextDispatch>,
```

Comment:
Since all of the variants here are either ZSTs or `Arc`s I'm fine with not boxing it.

Reply:
Ah right, I meant to remove the `Box` when I added the use of `Arc`s.